actors/builtin/reward/reward_state.go

```diff
@@ -20,6 +20,8 @@ type State struct {
 	BaselineSupply abi.TokenAmount // current supply
 
 	LastPerEpochReward abi.TokenAmount
+
+	LastKPIUpdate abi.ChainEpoch
```

Comment:
This should probably be initialized to -1 in ConstructState so that the first epoch doesn't ignore the reported power